tasks/win_agent_latest.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+- name: (Custom) Set agent download filename to latest
+  set_fact:
+    dd_download_url: "{{ datadog_windows_download_url }}"
+  when: datadog_windows_download_url | length > 0
+
+- name: Set agent download filename to latest
+  set_fact:
+    dd_download_url: "https://s3.amazonaws.com/ddagent-windows-stable/datadog-agent-{{ datadog_agent_major_version }}-latest.amd64.msi"
```

Comment:
We should use `datadog_windows_versioned_url` here

Reply:
The file names are different `datadog-agent-6-latest.amd64.msi` vs `ddagent-cli-X.Y.Z.msi` and having 2 different variables is useful (for example when we need to pin a specific version as latest). I'll say let's use 2 vars.

Reply:
To specify a specific version as latest, we already have the `datadog_windows_download_url` variable (that we could rename, it's not that explicit that it overrides the default latest), used above. Would that be enough? Otherwise we'll have 3 variables, with some overlap (one to override the full path to latest, one to override the path prefix to latest, and another one to override the path prefix to a versioned msi), which may be confusing to users.

Reply:
My main point here is to avoid having hard-coded information in the tasks. Using the defaults makes our role more flexible and easier to customize of users.

I think we only need 2:
- 1 to overwrite latest
- 1 for the versioned path